src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java

```diff
@@ -1073,7 +1073,7 @@ public static ItemColor leavesItemColor() {
             .initialProperties(() -> Blocks.STRIPPED_SPRUCE_LOG)
             .lang("Stripped Rubber Log")
             .blockstate((ctx, provider) -> provider.logBlock(ctx.get()))
-            .tag(BlockTags.MINEABLE_WITH_AXE)
+            .tag(ItemTags.LOGS_THAT_BURN, BlockTags.MINEABLE_WITH_AXE)
```

Comment:
that's an item tag so put it on the item, not the block

Reply:
should be fixed